framework/src/modules/chain/bft/bft.js

```diff
@@ -73,6 +73,7 @@ class BFT extends EventEmitter {
 	/**
 	 * Initialize the BFT module
 	 *
+	 * @param {Object} minActiveHeightsOfDelegates - Minimum active heights of a delegate
```

Comment:
Shouldn't it be `number` or maybe I'm mistaken. The same for the rest.  Also I think it's optional param

Reply:
No the structure is {[string]: number[] }

Reply:
Also actually it's not optional.